test/test_asn1.rb

```diff
@@ -566,6 +566,19 @@ def test_decode_constructed_overread
     assert_equal 17, ret[0][6]
   end
 
+  def test_constructive_each
+    data = ['test','data']
+    seq = OpenSSL::ASN1::Sequence.new data
+
+    assert_equal data, seq.entries
+  end
+
+  def test_constructive_nonarray_each
+    seq = OpenSSL::ASN1::Sequence.new 5
+
+    assert_raise(NoMethodError) { seq.each }
```

Comment:
Thanks for pointing that out. I was unaware of the code sharing with the Ruby tree. I removed this test in the latest commit.